Configuration/TypoScript/Libraries/setup.typoscript

```diff
@@ -0,0 +1,42 @@
+lib.fulltext = USER
+lib.fulltext {
+    userFunc = Kitodo\Dlf\Plugin\Toolbox->main
+    pages = 1
+    tools = tx_dlf_fulltexttool
+    solrcore = 1
+    fileGrpsImageDownload = DEFAULT,MAX
+    templateFile =
+}
+lib.fulltext < plugin.tx_dlf_toolbox
```

Comment:
I don't like the naming "Library" here. It's just a kitodo.Presentation plugin you configure at this point. In this case the tool tx_dlf_fulltexttool inside the toolbox plugin.

You could use the tools without the toolbox around. Than you can even drop this new TypoScript object.

For example the following should do the same (of course, you have to change the call in the fluid template as well):

```
plugin.tx_dlf_fulltexttool {
    pages = 1
    solrcore = 1
    fileGrpsImageDownload = DEFAULT,MAX
    templateFile = {$config.templateFileToolFulltext}
}
```

That's how we do in the DFG-Viewer an other pages.

https://github.com/slub/dfg-viewer/blob/master/Configuration/TypoScript/Plugins/kitodo.typoscript#L140

Reply:
> I don't like the naming "Library" here. It's just a kitodo.Presentation plugin you configure at this point. In this case the tool tx_dlf_fulltexttool inside the toolbox plugin.

I have moved code back to Plugins and remove Libraries directory.

> You could use the tools without the toolbox around. Than you can even drop this new TypoScript object.
>
> For example the following should do the same (of course, you have to change the call in the fluid template as well):
> 
> ```
> plugin.tx_dlf_fulltexttool {
>     pages = 1
>     solrcore = 1
>     fileGrpsImageDownload = DEFAULT,MAX
>     templateFile = {$config.templateFileToolFulltext}
> }
> ```

So for our case it would be something like that?
```
tx_dlf_fulltexttool {
    pages = {$constant.configPid}
    tools =
    solrcore = 1
    fileGrpsImageDownload = DEFAULT,MAX
    activateFullTextInitially = 1
    fullTextScrollElement = div#tx-dlf-fulltextselection
    searchHlParameters = hl,tx_dlf[highlight_word]
    templateFile = EXT:dlf/Resources/Private/Templates/FulltextTool.tmpl
  }
```
Then the whole lib stuff goes totally out?

And at the end changing `<f:cObject typoscriptObjectPath="lib.fulltext" />` in template with `<f:cObject typoscriptObjectPath="plugin.tx_dlf_fulltexttool" />` ?